### ISSUE TITLE
graphviz was not found on the system

### ISSUE THREAD

**Issue by azizur** [adrianvlupu/C4-Builder#4]
During installation (on MacOS) I am getting this message:
`graphviz was not found on the system. Downloading vizjs instead. See http://plantuml.com/vizjs. This may take a few minutes.`

This means it should work regardless of `graphviz` installed or not because `vizjs` is being installed. However, I am getting an error in the generated output site.

The `graphviz` requirement is not listed as a dependency. It would be useful to have this documented for new starters.

On Mac OS, it's simple homebrew command to install it:
`brew install graphviz`

**Comment by azizur** [comment]
I can confirm after installing `graphviz` via brew, it worked as expected.

**Comment by adrianvlupu** [comment]
Indeed, I need to update the docs and try to do a cold install and debug it. When I started this, all my PCs already had plantuml dependencies installed so that never crossed my mind.

**Comment by azizur** [comment]
Oh man, just hit this same issue today again!

**Comment by sidcoelho** [comment]
Installing graphviz with npm also has no effect.
I just can't install c4builder by any means.
